galette/templates/default/elements/logged_user.html.twig

```diff
@@ -28,7 +28,7 @@
         </div>
     {% else %}
         {% if not login.getCompactMenu() %}
-            <div class="{{ component_classes }}">
+            <nav class="{{ component_classes }}" aria-label="{{ _T('User menu') }}">
```

Comment:
I'm not convinced adding "anything menu" is really required (and also, that adds new strings to translate, I'd prefer to avoid).

Reply:
After some quick search, it seems required, yes. And for "User menu", is seems makes sense.
I still doubt for some labels ("Secondary menu" or "Top menu" for example: I'm not sure myself what in the UI it represents). I've nothing better to propose right now (and I have to leave), and I do not know if it's better to keep a "bad" label, or not add label at all.

Reply:
I searched a bit, see my newest comments ;)